splink/comparison_level_library.py

```diff
@@ -2,6 +2,11 @@
 
 from .comparison_level_creator import ComparisonLevelCreator
 from .dialects import SplinkDialect
+from .input_column import InputColumn
+
+
+def input_column_factory(name, splink_dialect: SplinkDialect):
+    return InputColumn(name, sql_dialect=splink_dialect.sqlglot_name)
```

Comment:
I think this may have been mentioned previously, but perhaps we could (in the future) do away with this and absorb this functionality into `InputColumn` (i.e. allow it's constructor to take a `str` OR a `SplinkDialect`).
Although maybe that makes a nasty dependency ??

Reply:
Yeah - agreed.  Once we're done with all the comparisons/comparison levels I'd like to circle back to the InputColumn and see if there's a refactor that makes this easier